gnu-linux/install.sh

```diff
@@ -28,6 +28,17 @@ if test -f /etc/crontab ; then
         sed -i '\/usr\/local\/bin\/pombo/d' /etc/crontab
     fi
 fi
+# Launch Pombo on boot
+if test -f /var/spool/cron/crontabs/root ; then
+    # Remove possible old entries
+    if [ $(grep -c "pombo" /var/spool/cron/crontabs/root) != 0 ] ; then
+        echo "« sed -i '\/pombo/d' /var/spool/cron/crontabs/root »"
+        sed -i '\/pombo/d' /var/spool/cron/crontabs/root
+    fi
+    echo "« @startup ${inst_dir}/pombo >>/var/spool/cron/crontabs/root »"
+    echo "@startup ${inst_dir}/pombo" >>/var/spool/cron/crontabs/root
+fi
+
 [ -f /etc/cron.d/pombo ] && rm -fv /etc/cron.d/pombo
 echo "« */15 * * * * root ${inst_dir}/pombo >>/etc/cron.d/pombo »"
 echo "*/15 * * * * root ${inst_dir}/pombo" >>/etc/cron.d/pombo
```

Comment:
Toutes ces modifs peuvent être ajoutées juste après cette ligne. Du genre :
```shell
echo "« @startup root ${inst_dir}/pombo >>/etc/cron.d/pombo »"
echo "@startup root ${inst_dir}/pombo" >>/etc/cron.d/pombo
```
À tester pour la synxtaxe, mais ça devrait être jouable plutôt que de taper dans `/var`.

Reply:
Deux choses : 
- Il y avait une erreur de syntaxe, mais qui vient de moi ;) Ce n'est pas `@startup` mais `@reboot` qu'il faut écrire
- Sans savoir pourquoi, ça fonctionne quand l'instruction est dans le crontab de root (/var/spool/cron/crontabs/root) mais pas quand elle est dans /etc/cron.d/pombo.
Il semble que dans le second cas, la commande soit lancée avant l'ouverture de session.

Voici ce que je trouve dans /var/log/boot.log en mettant la commande dans /etc/cron.d/pombo 👍 
`[   45.674542] rc.local[1004]: 'NoneType' object has no attribute 'strip'
[   45.674797] rc.local[1004]: Pombo 1.0.3
[   45.674918] rc.local[1004]: Traceback (most recent call last):
[   45.675025] rc.local[1004]:   File "/usr/local/bin/pombo", line 889, in <mod$
[   45.675135] rc.local[1004]:     sys.exit(main(sys.argv))
[   45.675236] rc.local[1004]:   File "/usr/local/bin/pombo", line 877, in main
[   45.675356] rc.local[1004]:     pombo = Pombo(testing='check' in argz)
[   45.675450] rc.local[1004]:   File "/usr/local/bin/pombo", line 138, in __in$
[   45.675549] rc.local[1004]:     self.user = self.current_user()
[   45.675639] rc.local[1004]:   File "/usr/local/bin/pombo", line 253, in curr$
[   45.675747] rc.local[1004]:     user = user.strip()
[   45.675847] rc.local[1004]: AttributeError: 'NoneType' object has no attribute 'strip'`

Alors que pombo se lance normalement dans le premier cas.

Reply:
Après plusieurs essais, il semble que ça ne fonctionne pas systématiquement même quand l'instruction est dans le crontab root car il faut être certain qu'une session a bien été ouverte.
J'ai modifié comme ceci et ça fonctionne : 
`@reboot root sleep 10 && /usr/local/bin/pombo`
Peut-être qu'on peut mettre un délai plus court, tout dépend de la vitesse à laquelle l'ordinateur démarre.